templates/frontOffice/default/assets/src/js/thelia.js

```diff
@@ -474,7 +474,7 @@ var pseManager = (function($){
                 $.ajax({type: "POST", data: $(this).serialize(), url: url_action,
                     success: function(data){
                         $(".cart-container").html($(data).html());
-                        $.ajax({url:"ajax/addCartMessage", data:{ product_id: product_id, pse_id: pse_id },
+                        $.ajax({url:url_site + "ajax/addCartMessage", data:{ product_id: product_id, pse_id: pse_id },
                             success: function (data) {
```

Comment:
This is not working, we cannot rely on url_site, which may be empty or wrong. Use a addCartMessage variable, which contains the URL (see below).